peer_mentoring/tests/test_search.py

```diff
@@ -0,0 +1,59 @@
+import io
+
+import pytest
+from django.core.files.base import ContentFile
+from django.urls import reverse
+from factories import UserFactory
+from groups.models import Avatar, Category, Group
+from PIL import Image
+
+
+@pytest.fixture
+def user(db):
+    return UserFactory()
+
+
+@pytest.fixture
+def group(db, avatar, category, user):
+    group = Group.objects.create(
+        title="Test Group",
+        avatar=avatar,
+        description="This is a test group",
+        category=category,
+        moderator=user,
+    )
+    group.members.add(user)
+    return group
+
+
+@pytest.fixture
+def avatar(db):
```

Comment:
seen this before, if you rename fixtures.py to conftest.py you can use the avatar fixture in this module, or is this one slightly different?

Reply:
Yes, it wasn't finding it otherwise, but corrected with the conftest.py